From 4eec6ce872ac46987efeb9b9cdab0ad78a563bdb Mon Sep 17 00:00:00 2001
From: Michael Zoran <mzoran@crowfest.net>
Date: Sat, 14 Jan 2017 21:46:04 -0800
Subject: [PATCH] ARM64: Use dwc_otg driver by default for USB.

If it breaks on anybody, they can use the standard device tree
overlays to switch back to the dwc2 driver.

Signed-off-by: Michael Zoran <mzoran@crowfest.net>
---
 arch/arm/boot/dts/bcm2710.dtsi | 7 -------
 1 file changed, 7 deletions(-)

--- a/arch/arm/boot/dts/bcm2710.dtsi
+++ b/arch/arm/boot/dts/bcm2710.dtsi
@@ -146,10 +146,3 @@
 	interrupts = <8>;
 };
 
-#ifdef RPI364
-&usb {
-	compatible = "brcm,bcm2835-usb";
-	reg = <0x7e980000 0x10000>;
-	interrupts = <1 9>;
-};
-#endif
